.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing
+This page contains the contributing guidelines.
+
+## Setup
+### prerequisite
+- Node.js installed. 
+ _Note: Node 6 or greater would be better for "best results"._
+- Package manager - [yarn](https://yarnpkg.com/en/docs/getting-started) or [npm](https://docs.npmjs.com/)
```

Comment:
I am using Lerna + Yarn workspaces in this repo, so yarn is required.

Reply:
yeah I was thinking to make that one as * preferred*.
Okay will change it

Reply:
done 👍